src/main/java/com/khutircraftubackend/auth/AuthenticationService.java

```diff
@@ -156,7 +156,7 @@ private void sendVerificationEmail(UserEntity user) {
         userRepository.save(user);
     }
 
-    private String getRandomNumber(){
+    public String getRandomNumber(){
```

Comment:
maybe better put it into some utility class?
it's been used in at least 2 scenarios now:
- email change (any time for registered users)
- email verification (at the time when user registers and any time later if verification failed before)